content/change_notes/2.3.0.md

```diff
@@ -0,0 +1,81 @@
+might cause errors, even if the element to click is visible to the end user because it fails technical validation. For those
+cases, it is now possible to avoid all technical checks and validation by passing the configuration object `{force: true}`
+to the click action.
+
+{{<highlight javascript>}}
+_click(_button("Click Me"), {force: true}); // clicks without validation
+{{</highlight>}}
+
+_Please note that a forced click might lead to false positives, as no validation takes place._  
+For more information, please have a look at the 
+[_click() API documentation](https://sakuli.io/apidoc/sakuli-legacy/interfaces/legacyapi.html#_click).
+
+### Improved CLI error messages
+We improved the error messages of the CLI output. You will now receive detailed information about syntax errors
+in test scripts as well as more detailed information on the root cause for errors. 
+
+### Optional website stabilization
+Many modern websites use visual effects regularly. Those effects have impaired some automation during the last months.
+Some users compensated such situations by using `_wait()` with fixed times to wait. Depending on the system the checks 
+run on, we have observed that a strict `_wait()` is not reliable. Therefore we have added `_pageIsStable()` to the
+Sakuli API. `_pageIsStable()` allows you to wait until the DOM of your website has stabilized e.g. after animations or
+dynamic content, loaded via JavaScript. As soon as the DOM is stable, test execution continues. It is possible to
+configure the maximum timeout to wait and even the intervals in which the DOM is checked.
+
+{{<highlight javascript>}}
+// Wait for the DOM to stabilize within a maximum 5 seconds and check in an interval of 100 milliseconds.
+await _pageIsStable(5000, 100);
+{{</highlight>}}
+
+For more information, please have a look at the
+[_pageIsStable() API documentation](https://sakuli.io/apidoc/sakuli-legacy/interfaces/commonactionsapi.html#_pageisstable).
+
+## Enterprise packages
+### Sakuli Prometheus forwarder
+With 2.3.0, we have added a prometheus forwarder to the enterprise forwarder packages. To use the prometheus forwarder,
+a valid Sakuli enterprise license is required. As Sakuli checks are not designed to provide a persistent endpoint for
+Prometheus to scrape, the forwarder leverages the prometheus push gateway. For more information about
+configuration and setup, please have a look at the
+[prometheus forwarder documentation](https://sakuli.io/docs/enterprise_features/prometheus/).
+
+### CheckMK forwarder - Website link in spool files
+From now on, Sakuli 2 adds an optional website URL to the CheckMK spool files as it did in Version 1.x.x . These URLs
+will then be displayed in the CheckMK Web Interface which allows an easy access for the monitoring staff to systems 
+under test in case of an error. For more information, please have a look at the
+[CheckMK forwarder documentation](https://sakuli.io/docs/enterprise_features/check/#forwarder-configuration).
```

Comment:
Sakuli 1 did not provide this functionality out of the box. It was only possible due to @tnobody's hacking skills 😃 

```suggestion
From now on, Sakuli allows to add an optional website URL to the CheckMK spool files. These URLs will be displayed in the CheckMK Web Interface which allows an easy access for the monitoring staff to systems under test in case of an error.

For more information, please have a look at the [CheckMK forwarder documentation](https://sakuli.io/docs/enterprise_features/check/#forwarder-configuration).
```

Reply:
Ah, good to know.